dotcom-rendering/src/web/browser/hydration/init.ts

```diff
@@ -2,11 +2,11 @@ import '../webpackPublicPath';
 
 import { startup } from '@root/src/web/browser/startup';
 import { log } from '@guardian/libs';
-import { whenVisible } from './whenVisible';
-import { whenIdle } from './whenIdle';
+import { whenVisible } from '../whenVisible';
+import { whenIdle } from '../whenIdle';
 import { doHydration } from './doHydration';
-import { getName } from './getName';
-import { getProps } from './getProps';
+import { getName } from '../getName';
+import { getProps } from '../getProps';
```

Comment:
I moved these to the common root folder so they can be shared. We can consider a different folder structure, this just worked for now